### ISSUE TITLE
BUG: Close settings button changes toggle option (extremely minor issue)

### ISSUE THREAD

**Issue by stroiman** [zsviczian/obsidian-excalidraw-plugin#2085]
### Have you searched for existing issues (including closed ones)?

- [X] Yes, I have searched the existing issues.

### Does this bug persist in a new vault with only Excalidraw installed?

- [X] Yes, I have verified the issue persists.

### Your environment

SYSTEM INFO:
	Obsidian version: v1.7.4
	Installer version: v1.5.3
	Operating system: Darwin Kernel Version 23.6.0: Mon Jul 29 21:13:04 PDT 2024; root:xnu-10063.141.2~1/RELEASE_ARM64_T6020 23.6.0
	Login status: not logged in
	Language: en
	Insider build toggle: off
	Live preview: on
	Base theme: adapt to system
	Community theme: none
	Snippets enabled: 0
	Restricted mode: off
	Plugins installed: 1
	Plugins enabled: 1
		1: Excalidraw v2.6.2

RECOMMENDATIONS:
	Community plugins: for bugs, please first try updating all your plugins to latest. If still not fixed, please try to make the issue happen in the Sandbox Vault or disable community plugins.

### Describe the bug

First of all, huge amount of thanks for making the excalidraw plugin. I have nothing but respect for people who manage to deliver such excellent software in their spare time.

Which is why I really don't think you should bother with this, as this is just the tiniest of issues; but I noticed it, so I'll report it.

At first thought this was an Obsidian bug, but I didn't experience the same for another plugin.

When the settings is open, and scrolled so that a toggle button is on height with the setting dialog's close button, clicking the X switches the toggle; rather than closing settings.

For example, clicking the X here toggles "Enable single-finger panning in pen mode":

<img width="895" alt="image" src="https://github.com/user-attachments/assets/230cb4b8-1e5b-44ac-b302-85ae4d7dbb34">

Yeah, totally not important, I know.

**Comment by zsviczian** [comment]
That is an Obsidian issue, not an Excalidraw issue, Excalidraw simply create a Setting's form object that Obsidian renders... but an issue nonetheless

**Comment by stroiman** [comment]
Allright, then my apoligies for bothering you.

As mentioned, I didn't see it in other plugins, but I'll check if I can replicate with other plugins - or possibly create a dummy plugin to replicate the issue and report to Obsidian instead.

**Comment by zsviczian** [comment]
Good point... I always just assumed this was an issue across the board, but no, this is Excalidraw specific. I will explore in more detail.

**Comment by zsviczian** [comment]
Found the css causing this. 

changed it from 
```css
.excalidraw-settings input:not([type="color"]) {
  min-width: 10em;
}
```

to
```css
.excalidraw-settings input[type="text"] {
  min-width: 10em;
}
```

The objective is/was to make the text input fields at least 10 characters long.

Thanks for raising and for being persistent. 

Will release the fix with 2.6.3

**Comment by stroiman** [comment]
Cool,

Are you aware of the `width` property if `<input>` elements? Sounds like it _may_ be a better alternative. but it's a direct attribute, not a style setting, so it can't be controlled through CSS (at least that's what my instincts tell me).

But generally sets the size of the field. The field can grow, but refuse to shrink any smaller (which is confusing as hell when you don't know why and try to fix a layout for mobile).
